fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/service/LoanApplicationWritePlatformServiceJpaRepositoryImpl.java

```diff
@@ -384,6 +393,76 @@ public CommandProcessingResult submitApplication(final JsonCommand command) {
             }
 
             this.loanRepositoryWrapper.save(newLoanApplication);
+            // BigDecimal totalCollateralValue = new BigDecimal(0);
+
+            /**
+             * TODO: Update `m_exchange_loan_collateral` table
+             */
+
+            // boolean isUpdatable = isLoanCollateralUpdatable(command);
+            //
+            // if (isUpdatable) {
+            // Set<LoanCollateralManagement> loanCollateralManagementSet = new HashSet<>();
+            // Set<ClientCollateralManagement> clientCollateralManagementSet = new HashSet<>();
+            //
+            // // Extract array of collaterals as a string array
+            // String[] collaterals = this.fromJsonHelper.extractArrayNamed("collaterals", command.parsedJson());
+            //
+            // LoanCollateralManagement loanCollateralManagement = null;
+            // BigDecimal quantity = BigDecimal.valueOf(0);
+            //
+            // for (String collateral : collaterals) {
```

Comment:
why is this still here?

Reply:
Missed those. Since this contains large number of files.